code/my_set/model/my_set.py

```diff
@@ -0,0 +1,92 @@
+class MySet:
+    # The class sets (integer set) using lists
+    def __init__(self, elements=None):
+        self.my_set = list()
+        if isinstance(elements, list):
+            self.my_set = elements
+        elif elements is None:
+            pass
+        elif isinstance(elements, int):
+            self.my_set.append(elements)
+        else:
+            raise TypeError('Input error: argument type must be list')
+
+    def get_size(self):
+        return len(self.my_set)
+
+    def is_empty(self):
+        return self.get_size() == 0
+
+    def __contains__(self, element):
+        if element in self.my_set:
+            return True
+        return False
+
+    def __iter__(self):
+        for element in self.my_set:
+            yield element
+
+    def __eq__(self, obj):
+        if isinstance(obj, list):
+            return self.my_set == obj
+        elif isinstance(obj, MySet):
+            return self.my_set == obj.my_set
+        elif isinstance(obj, int):
+            if (self.get_size() != 1):
+                return False
+            else:
+                return self.my_set[0] == obj
+        else:
+            raise TypeError('Input error: argument type must be list or int or another set')
+
+    def add(self, element):
+        if element not in self.my_set:
+            self.my_set.append(element)
+
+    def delete(self, element):
+        if element not in self.my_set:
+            raise ValueError('Set doesnt contain this element')
+        else:
+            position = self.my_set.index(element)
+            del self.my_set[position]
+
+    def union(self, obj):
+        if isinstance(obj, list):
+            for element in obj:
+                self.add(element)
+        elif isinstance(obj, MySet):
+            self.union(obj.my_set)
```

Comment:
This part can be simplified
```suggestion
        if isinstance(obj, (list, MySet)):
            for element in obj:
                self.add(element)
```